pkg/packagekit/updates.css

```diff
@@ -38,8 +38,11 @@ tr.security {
 }
 
 /* don't let the install progress bar get too wide */
-.progress {
+.progress-main-view {
     max-width: 60rem;
+    margin-top: 10ex;
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
If you don't mind setting the margin for the bottom (in this example to 0, but we might want to have something different), these could be collapsed into:

```css
margin: 10ex auto 0;
```

Shorthand for `top sides bottom`.